### PR TITLE
MITRE notebook bugfixes and updated navigation layer files

### PR DESCRIPTION
**Pull request by ashwin-patil**
Fixes #

## Proposed Changes

  - Added bugfixes for notebook
  - sentinel rebranding 
  - updated output csv files
  - generated new navigator layer files with attack v10 - navigator - 4.5.2
  - updated svg files

### CODE REVIEW

**Comment by ashwin-patil** [comment]
@aprakash13  sampleDataValidator test is failing for Navigator json layer files which are in different format as compared to traditional log files. Not sure how to add exception for these or recommended way to pass these tests. 

Let me know and will update PR accordingly.

**Comment by preetikr** [comment]
@chinguyen1 - to help review notebooks included 
@ashishsyal  / @v-jayakal - to review everything else 

Thanks!

**Comment by chinguyen1** [comment]
Assigning notebook review to @ianhelle. Can you please help review the notebook?

**Comment by vmanojreddy** [comment]
Logo should be less than 5kb.

**Comment by petebryan** [comment]
So on the notebooks there are a few changes you can make:
init_notebook can be accessed directly msticpy now.
i.e.
```
import msticpy
msticyp.init_notebook()
```

You can remove the commented out widgets import now

Could we move the items defined as 'Custom dataset for non-standard formatted alerts in the doc. - May need updates to keep it current' to a table on GitHub to make version control and updates easier?

There are some other re-factoring elements you could also consider to reduce the amount of code needed but these aren't things that need to be done now.

**Review by vmanojreddy — CHANGES_REQUESTED** [review]

File: Sample Data/MITRE ATT&CK/images/AWS_Layer_Json_File_for_Azure_Sentinel.svg [hunk 1]
```diff
@@ -1,2 +0,0 @@
...
...e</tspan></text></g><g class="technique T1498" transform="translate(0, 366.3636363636364)"><rect class="cell" height="73.27272727272728" width="95.09090909090908" fill="white" stroke="#6B7279"/><polygon class="sidebar" transform="translate(0, 73.27272727272728)" points="0,0 12,0 12,12" fill="#6B7279" visibility="hidden"/><text font-size="12" fill="black"><tspan x="4" y="25.469696969696972">Network</tspan><tspan x="4" y="40.13636363636364">Denial</tspan><tspan x="4" y="54.803030303030305">of Service</tspan></text></g><g class="technique T1496" transform="translate(0, 439.6363636363636)"><rect class="cell" height="73.27272727272728" width="95.09090909090908" fill="#8ec843" stroke="#6B7279"/><polygon class="sidebar" transform="translate(0, 73.27272727272728)" points="0,0 12,0 12,0" fill="#6B7279" visibility="hidden"/><text font-size="12" fill="black"><tspan x="4" y="32.88636363636364">Resource</tspan><tspan x="4" y="47.38636363636364">Hijacking</tspan></text></g></g><g class="subtechniques"/><g class="tactic-label"><text font-size="12" fill="black" font-weight="bold"><tspan text-anchor="middle" x="47.54545454545454" y="40.13636363636364">Impact</tspan></text></g></g></g></g></g></svg>
```
[the remark this answers is not in the document]
Comment (ashwin-patil) [reply Sample Data/MITRE ATT&CK/images/AWS_Layer_Json_File_for_Azure_Sentinel.svg:2]: Unfortunately this is not possible. These are image file formats and it is required to display images correctly. We may have to add exception for such file formats in the failing tests.

**Comment by ashwin-patil** [comment]
> Logo should be less than 5kb.

There are no logos in this PR. SVG files are referenced in Workbook as static representation of MITRE Matrix hence included in the Sample data folder. We may need some changes to tests to add exception for it.

**Comment by vmanojreddy** [comment]
@ashwin-patil  Sample data file data must be in the form of array.

**Comment by vmanojreddy** [comment]
@ashwin-patil  Please resolve conflicts.

**Comment by ashwin-patil** [comment]
> @ashwin-patil Sample data file data must be in the form of array.

> @ashwin-patil Please resolve conflicts.

Clean PR #4221 is opened , closing this old PR. Please review new PR
Changes/Fixes: 
- resolve merge conflicts
- notebook updates.
- passing all validation checks

**Pull request closed without merging**